fastlane/spec/actions_specs/carthage_spec.rb

```diff
@@ -423,6 +443,20 @@
         end.not_to raise_error
       end
 
+      it "works with cache_builds" do
+        expect do
+          Fastlane::FastFile.new.parse("lane :test do
+            carthage(
+              use_ssh: true,
+              use_submodules: true,
+              use_binaries: false,
+              cache_builds: true,
+              platform: 'iOS'
+            )
+          end").runner.execute(:test)
+        end.not_to raise_error
```

Comment:
This is consistent with the tests around it, so we should ship it, IMO. If folks feel strongly, we can always take it back out. 👍